Zinc/Private/ZincRepoIndex.m

```diff
@@ -226,6 +226,12 @@ - (void) removeBundle:(NSURL*)bundleResource
     }
 }
 
+- (void) purgeBundleWithID:(NSString *)bundleID {
```

Comment:
Different naming makes sense. 
for the `delete` method, are you referring to `removeBundle` or do we have yet another method that sounds similar?